buildtest/cli/commands.py

```diff
@@ -0,0 +1,29 @@
+def list_buildtest_commands():
+    """This method implements command ``buildtest commands`` which shows a list
```

Comment:
Let's just make this one line. I see this statement broken up in 3 lines. The sentence is not that long for it to require new line.

Reply:
I got a "weak warning" about this previously, I am correcting this now.